rs/https_outcalls/consensus/src/gossip.rs

```diff
@@ -72,6 +72,8 @@ impl<Pool: CanisterHttpPool> PriorityFnFactory<CanisterHttpResponseShare, Pool>
                 return Priority::Drop;
             }
 
+            // We derive the highest accepted request id from the next expected request id, plus the
+            // number of maximal number of requests we can have in-flight.
```

Comment:
Isn't the "maximal number of requests we can have in-flight" a different constant, i.e. the one we introduced [here](https://github.com/dfinity/ic/pull/859/files#diff-4a98637a515a89b30d41c31d4f9207d3a0f8ba0e1af7f1cfa880baaf3051a2a3R141)?

Reply:
Yes, this comment is outdated, thanks for the catch!